src/components/pseudonym/AnonymousConsentGate.styles.scss

```
@@ -65,15 +65,15 @@
 	font-size: 32px;
 	line-height: 1.2em;
 	font-weight: 600;
-	color: #1f2937;
+	color: var(--m3-on-surface, #1f2937);
 	margin: 0;
 }
 
 .anonymousConsentGate__body {
 	font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)